x-pack/metricbeat/module/aws/elb/manifest.yml

```diff
@@ -0,0 +1,7 @@
+default: true
+input:
+  module: aws
+  metricset: cloudwatch
+  defaults:
+    metrics:
+      - namespace: AWS/ELB
```

Comment:
for my previous comment, that would mean we define the metrics we want here :)

Reply:
Yep, I added 
```
defaults:
    metrics:
      - namespace: AWS/ELB
        statistic: ["Average"]
        tags.resource_type_filter: elasticloadbalancing
```

Reply:
This is a good description of all available metrics, I see there are different statistics recommended depending on the metric, this is a good opportunity to leverage your latest updates here! https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-cloudwatch-metrics.html

I'm curious, I see many of these are not present in `data.json`, do you know why?

Reply:
Good point! I will change it 😬 to adapt different statistic methods. A lot of the metrics have no value at the time when `data.json` is generated. For example, here is a screenshot from cloudwatch for ELB latency metric:
<img width="1400" alt="Screen Shot 2019-08-01 at 8 42 04 AM" src="https://user-images.githubusercontent.com/14081635/62302916-6ceab480-b438-11e9-954c-0cf3f0cd29e8.png">